herbert_core/utilities/classes/@object_lookup/object_array.m

```diff
@@ -25,5 +26,20 @@
 end
 
 % Get return argument
+if numel(varargin)==1
+    iarray = varargin{1};
+    if ~isscalar(iarray)
+        error('Index to original object array, ''iarray'', must be a scalar')
+    end
+elseif numel(varargin)==0
```

Comment:
Why not `isempty`?

Reply:
Sheer force of habit. I think probably because line 29 tested if numel(varargin)==1 and this was the test if ==0. Does it make any real difference?

Reply:
No, not really. Apparently it doesn't even flag as an mlint issue which I thought it would.